tf/scripts/frontend_user_data.sh

```
@@ -18,3 +18,15 @@ sudo docker run -d -p 3000:3000 lti-frontend
 
 # Timestamp to force update
 echo "Timestamp: ${timestamp}"
+
+
+# Instalar el agente de Datadog
+DD_API_KEY=${datadog_api_key} DD_SITE="us5.datadoghq.com" bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script.sh)"
```

Shellcheck (0.11.0)

[warning] datadog_api_key is referenced but not assigned.

(SC2154)